db/migrate/20250617145428_add_guardian_individual_tax_id_to_user_compliance_info.rb

```
@@ -0,0 +1,5 @@
+class AddGuardianIndividualTaxIdToUserComplianceInfo < ActiveRecord::Migration[7.1]
```

RuboCop (1.75.5)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)